themes/mlibrary_new/exhibit-builder/exhibits/summary.php

```diff
@@ -11,65 +11,73 @@
   );
 
 ?>
+<!--Breadcrumb and Share Bar-->
+  <div class="col-xs-9">
+      <ol class="breadcrumb">
+        <li class="breadcrumb-item"><a href="#">Home</a></li>
```

Comment:
use link_to_home_page(__('Home'))

Reply:
I still have to work on the breadcrumbs and their styling. That will be a separate PR. Also, since we are using breadcrumbs in a variety of pages, we should use a function to build it in the pages.

Reply:
This function will be used on all pages to add the Home link. replace line 17 <a href="#">Home</a> with link_to_home_page(__('Home')). It will auto generate the link with url.

Reply:
Ok, I did what you requested.